wallet/src/de/schildbach/wallet/service/BlockchainServiceImpl.java

```diff
@@ -111,6 +111,7 @@
 import android.os.PowerManager;
 import android.os.PowerManager.WakeLock;
 import androidx.core.app.NotificationCompat;
+import androidx.lifecycle.LifecycleService;
```

Comment:
This is interesting.  Perhaps later we can upgrade BlockchainService based on Bitcoin Wallet 6.x+

Reply:
Would be awesome. They replaced some broadcasts with more convenient LiveDatas.